Objects/genericaliasobject.c

```diff
@@ -588,6 +613,13 @@ setup_ga(gaobject *alias, PyObject *origin, PyObject *args) {
     alias->args = args;
     alias->parameters = NULL;
     alias->weakreflist = NULL;
+
+    if (PyVectorcall_Function(origin) != NULL) {
+        alias->vectorcall = ga_vectorcall;
+    } else {
+        alias->vectorcall = ga_make_tp_call;
```

Comment:
Is there any benefit to using `ga_make_tp_call` instead of just leaving it as `NULL`? IIUC, [that's what functools.partial does](https://github.com/python/cpython/blob/3ae975f1ac880c47d51cca6c9e305547bd365be7/Modules/_functoolsmodule.c#L268-L282).

Reply:
Answering my own question, the difference is avoiding checking for the presence of a vectorcallfunc and then removing a recursion check.